mailpoet/assets/js/src/landingpage/faq.tsx

```diff
@@ -0,0 +1,128 @@
+import ReactStringReplace from 'react-string-replace';
+import { __ } from '@wordpress/i18n';
+import { Heading } from 'common/typography/heading/heading';
+
+function Faq() {
+  const list: {
```

Comment:
Defining these types under a separate named TS `type` would make it easier to re-use and read, what do you think?

Reply:
I don’t believe we will be reusing this type anywhere else. Extracting it into a named `type` adds verbosity to the codebase.

Do let me know if you feel strongly about it and still want me to go ahead and add the type.

Reply:
It's fine to keep it like this as well.